.gitignore

```diff
@@ -0,0 +1,77 @@
+
+# Created by https://www.toptal.com/developers/gitignore/api/c++,meson,ninja,linux,visualstudiocode
+# Edit at https://www.toptal.com/developers/gitignore?templates=c++,meson,ninja,linux,visualstudiocode
+
+### C++ ###
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+*.smod
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
+
+### Linux ###
+*~
+
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
+
+### Meson ###
+# subproject directories
+/subprojects/*
+!/subprojects/*.wrap
+
+### Ninja ###
+.ninja_deps
+.ninja_log
+
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
```

Comment:
We do not use vscode, so I do not think we need these exceptions in the .gitignore ..

Reply:
Fixed.